src/smtpclient.cpp

```diff
@@ -369,34 +377,109 @@ bool SmtpClient::sendMail(MimeMessage& email)
             sendMessage("RCPT TO:<" + (*it)->getAddress() + ">");
             waitForResponse();
 
-            if (responseCode != 250) return false;
+            switch(responseCode) {
+                case 250:
+                case 251:
+                    break;
+                case 550:
+                case 551:
+                case 553:
+                case 555:
+                    slVerify.clear();
+                    emit smtpError(ClientError, responseCode, (*it)->getAddress());
+                    slVerify = Verify((*it)->getAddress());
+                    if(slVerify.size() > 0) {
+                        for(int inConta = 0; inConta < slVerify.size(); inConta++) {
+                            sendMessage("RCPT TO:<" + slVerify.at(inConta).simplified() +">" );
+                            waitForResponse();
+                            if(responseCode != 250) {
+                                break;
+                            }
+                        }
+                    }
+                    break;
+                default:
+                    emit smtpError(ClientError, responseCode);
+                    return -2000 - responseCode;
+            }
         }
 
         // Cc (carbon copy)
         for (it = email.getRecipients(MimeMessage::Cc).begin(), itEnd = email.getRecipients(MimeMessage::Cc).end();
              it != itEnd; ++it)
         {
-            sendMessage("RCPT TO:<" + (*it)->getAddress() + ">");
+            sendMessage("RCPT CC:<" + (*it)->getAddress() + ">");
```

Comment:
`RCPT CC` and `RCPT BCC` does not seems to be standard SMTP commands. Normally, each recipient is added with the `RCPT TO` command, while the CC and BCC lists are controlled in the MIME headers.

Can you add more details about why you need these commands?

Reply:
Because if they do not use BCC, they can appear in TO (message header) list...
In some cases, this is a security breach.
I will test your solution and talk again...

Reply:
If they do not use BCC, the addresses may appear in TO (message header) list.
This could be a security breach.
I will test your solution and we can talk again...